common/lib/xmodule/xmodule/vertical_module.py

```diff
@@ -1,23 +1,34 @@
+"""
+VerticalBlock - a pure XBlock.
+"""
```

Comment:
What do you think about moving this file to be `vertical_block.py`? I didn't do it initially, figuring we could do it at the end if we wanted.